usaspending_api/download/management/commands/restart_download.py

```diff
@@ -0,0 +1,63 @@
+import logging
```

Comment:
Is it possible just to combine these two commands into one since they both restart "frozen" jobs. For instance just have another `download_job_id` param that, when passed in, targets only that job. 

From a "operator" perspective, this would be easier to get the same job done with one command

Reply:
It's possible. But making them separate kept them smaller and easier to understand